src/TomanPay/senior_backend_position.md

```diff
@@ -0,0 +1,65 @@
+# [Toman](https://tomanpay.net/)
+
+### Status
+#### 📜📞🔧❌
+## Senior Backend Developer (Python/Django)
+### Interview process
+```mermaid
+flowchart LR
+    sr(Send resume) --> hr(HR call) --> ti1(1st Technical Interview) --rejected--x  ti2(2nd Technical Interview) -.-> hri(HR Interview) -.-> o(Offer)
+```
+### Apply way
+Refer
+
+### Interview date
+- **Refered** <br /> Unknown!
```

Comment:
We don't need this line. Delete it.

Reply:
use please after "it". We need it. thanks.

Reply:
=))